v2/googlecloud-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/templates/spannerchangestreamstobigquery/TypesUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I say we would need more tests here to improve the coverage of TypesUtils since it does a lot of String manipulation

Reply:
Done. I added tests to cover all string manipulation branches in`TypesUtils.java` .